src/shogun/machine/StochasticGBMachine.cpp

```diff
@@ -260,10 +257,11 @@ CMachine* CStochasticGBMachine::fit_model(CDenseFeatures<float64_t>* feats, CReg
 	return c;
 }
 
-CRegressionLabels* CStochasticGBMachine::compute_pseudo_residuals(CRegressionLabels* inter_f)
+CRegressionLabels* CStochasticGBMachine::compute_pseudo_residuals(
+    CRegressionLabels* inter_f, CLabels* labs)
 {
-	REQUIRE(m_labels,"training labels not set!\n")
-	SGVector<float64_t> labels=(dynamic_cast<CDenseLabels*>(m_labels))->get_labels();
```

Comment:
pls avoid such whitespace changes in PRs.
They should be in a sep PR. It just pollutes the diff (i.e. hard to review, takes longer)